web/angular-wallet/proxy.config.json

```diff
@@ -0,0 +1,12 @@
+{
+  "/burst": {
+    "target": {
+      "host": "3.16.150.48",
```

Comment:
intentional?

Reply:
Actually,  I don't like to put any IP or similar info into the repo.  Should be removed. Agree. I did something similar for the e2e test setup.